src/Arcus.Scripting.ApiManagement/Arcus.Scripting.ApiManagement.psd1

```diff
@@ -24,7 +24,7 @@ Author = 'Arcus'
 CompanyName = ''
 
 # Copyright statement for this module
-Copyright = '(c) 2020 Arcus. All rights reserved.'
+Copyright = '(c) 2022 Arcus. All rights reserved.'
```

Comment:
We can maybe look into generating this, and/or removing it. I don't think it's added to the other libraries.